database/db.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+function connectDB(){
+  $host = '151.80.32.16:6033';
+  $user = 'if0_35606101';
+  $password = 'UT57Hj7nKcA';
+  $database = 'if0_35606101_garage_train';
```

Comment:
DO not never put variable into code (Credentials should not be hard-coded : CRITICAL)